Backend/src/domain/Game.ts

```diff
@@ -10,21 +10,42 @@ export class Game {
     private readonly status: GameStatus;
 
     constructor(id: number, guessedLetters: string[], wrongLetters: string[],
-        word: string, hiddenWord: string, guesses: number) {
+        word: string) {
         this.id = id;
         this.guessedLetters = guessedLetters;
         this.wrongLetters = wrongLetters;
         this.word = word;
-        this.hiddenWord = hiddenWord;
-        this.guesses = guesses;
+        this.hiddenWord = this.hideWord();
+        this.guesses = this.getGuessesCount();
         this.status = this.decideGameStatus();
     }
 
+    private getGuessesCount(): number {
+        return this.guessedLetters.length + this.wrongLetters.length;
+    }
+
+    private hideWord(): string {
+        return Array.from(this.word).map(letter => {
+            if (this.inGuessedLettersIsThisLetter(letter))
+                return letter
+            else
```

Comment:
šito else nereikia, tau ir taip pakvies tą this.isWhiteSpace(letter) ? ' ' : '#', nes pirmo ifo vidus turi returną